packages/proxy/lib/http/response-middleware.ts

```diff
@@ -509,6 +509,9 @@ const MaybeSendRedirectToClient: ResponseMiddleware = function () {
 
 const CopyResponseStatusCode: ResponseMiddleware = function () {
   this.res.status(Number(this.incomingRes.statusCode))
+  // Set custom status message/reason phrase from http response
+  // https://github.com/cypress-io/cypress/issues/16973
+  this.res.statusMessage = this.incomingRes.statusMessage || ''
```

Comment:
let's keep whatever the default is

```suggestion
  this.res.statusMessage = this.incomingRes.statusMessage || this.res.statusMessage
```

Reply:
or better yet

```suggestion
  if (this.incomingRes.statusMessage) this.res.statusMessage = this.incomingRes.statusMessage
```

Reply:
> or better yet

This is done. Thank you for the suggestion @flotwig